plugins/missions.js

```diff
@@ -190,17 +194,124 @@ window.plugin.missions = {
 		}
 	},
 
-	showMissionListDialog: function(missions) {
-		dialog({
-			html: this.renderMissionList(missions),
-			height: 'auto',
-			width: '400px',
-			collapseCallback: this.collapseFix,
-			expandCallback: this.collapseFix,
-		}).dialog('option', 'buttons', {
-			'Create new mission': function() { open('//missions.ingress.com'); },
-			'OK': function() { $(this).dialog('close'); },
-		});
+	showMissionListDialog: function(missions, caption, isPortalList) {
+
+		this.isShowingPortalList = isPortalList;
+
+		// Check whether dialog is already open
+		var openDialog = window.DIALOGS['dialog-missionsList'];
+		if (openDialog) {
+			// Dialog already there, expand if collpased
+			// NOTE: It would be really cool if the IITC core had some function to read and modify collpased state of dialogs
+			// but it hasn't so we need to access internal information here
```

Comment:
Do we need this comment here?
And are you sure that we can't  access internal info?